package.json

```diff
@@ -39,8 +39,8 @@
     "@svgr/webpack": "6.2.1",
     "autoprefixer": "10.2.6",
     "babel-jest": "26.6.3",
-    "babel-loader": "8.2.3",
-    "babel-plugin-react-intl": "7.9.4",
+    "babel-loader": "^8.2.5",
+    "babel-plugin-formatjs": "^10.3.27",
```

Comment:
I see a few breaking changes:

* https://github.com/formatjs/formatjs/commit/635633d9aa6db0c02e340c96cb56fffe2cde54ee: are we ok with lang files changing IDs?  I'm not sure myself.
* https://github.com/formatjs/formatjs/commit/e310baf561b0313f251d16e9b97109cd40cbcf0e: I comment above on the removal of moduleSourceName
* https://github.com/formatjs/formatjs/commit/47ca556c1405eec7b7b2660275ae84a019112b2b: do we need to import `formatjs/cli` and/or make changes to Makefiles anywhere?

Reply:
i think these changes are not breaking anything. We have tested this branch in multiple consumers, specially after changing languages and no errors were found.
We also don't explicitly need to import `formatjs/cli` or make changes in Makefiles because we are not using it directly